examples/hello-world/hello-pt/jobs/hello-pt/app/custom/cifar10validator.py

```diff
@@ -48,6 +53,11 @@ def __init__(self, data_path="~/data", validate_task_name=AppConstants.TASK_VALI
         test_data = CIFAR10(root=data_path, train=False, transform=transforms)
         self._test_loader = DataLoader(test_data, batch_size=4, shuffle=False)
 
+    def handle_event(self, event_type: str, fl_ctx: FLContext):
+        if event_type == EventType.START_RUN:
```

Comment:
why do we move into "START_RUN" event?

Reply:
Because we want each client to store the download data to its unique location, which will be under the client_name (fl_ctx.get_identity_name()). Otherwise, concurrent clients will overwrite each other in the same location. This client_name is getting from the fl_ctx, only available when START_RUN.